$OpenBSD: patch-configure,v 1.10 2016/06/13 09:12:40 landry Exp $

- no Werror, breaks with gcrypt deprecation warnings
- dont check for pcap-bpf.h, not installed on openbsd
- in rrdtool the main librrd.so is now threadsafe, there is no more librrd_th.so

--- configure.orig	Fri Jan 22 10:55:47 2016
+++ configure	Mon Jun  6 09:18:40 2016
@@ -17470,7 +17470,7 @@ SAVE_CFLAGS="$CFLAGS"
 # Emulate behavior of src/Makefile.am
 if test "x$GCC" = "xyes"
 then
-	CFLAGS="$CFLAGS -Wall -Werror"
+	CFLAGS="$CFLAGS -Wall"
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for strtok_r" >&5
@@ -25194,7 +25194,7 @@ then
 
 
 fi
-if test "x$librrd_threadsafe" = "xyes"
+if 1
 then
 
 $as_echo "#define HAVE_THREADSAFE_LIBRRD 1" >>confdefs.h
